storybook/src/stories/Forms/VpnServerForm.stories.js

```diff
@@ -0,0 +1,301 @@
+      type: { required: true }, // required prop or not
+    },
+    "data.security_groups": {
+      description:
+        "An array of string values representing the security groups for the VPN Server.",
+      control: "none",
+      type: { required: true },
+    },
+    "data.client_dns_server_ips": {
+      description:
+        "A comma separated list (string) of the DNS server addresses that will be provided to VPN clients connected to this VPN server.",
+      control: "none",
+      type: { required: false }, // required prop or not
+    },
+    isModal: {
+      description: "Boolean value determining if the form is a modal",
+      type: { required: true }, // required prop or not
+      table: { defaultValue: { summary: false } },
+      control: "none",
+    },
+    vpnServerRouteProps: {
+      description:
+        "Passes through props from the VpnServerRouteForm, look at the VpnServerRouteForm story for its required props",
+      type: { required: true }, // required prop or not
+      control: "none",
+    },
+    "vpnServerRouteProps.onSubmit": {
+      description:
+        "A function that defines what occurs when a modal is submitted",
+      type: { required: true },
+    },
+    "vpnServerRouteProps.onSave": {
+      description: "A function that defines what occurs when the form is saved",
+      type: { required: true },
+    },
+    "vpnServerRouteProps.onDelete": {
+      description:
+        "A function that defines what occurs when the resource is deleted",
+      type: { required: true },
+    },
+    "vpnServerRouteProps.disableSave": {
+      description:
+        "A function that returns a single boolean describing whether the save button should be disabled",
+      type: { required: true },
+      control: "none",
+    },
+    invalidCallback: {
+      description: "Function that determines invalid state for `name` field",
+      type: { required: true }, // required prop or not
+      control: "none",
+    },
+    invalidTextCallback: {
+      description: "Function that determines invalid text for `name` field",
+      type: { required: true }, // required prop or not
+      control: "none",
+    },
+    resourceGroups: {
+      description:
+        "An array of strings containing the names of resource groups to select",
+      type: { required: true }, // required prop or not
+      control: "none",
+    },
+    vpcList: {
+      description: "An array of strings containing the names of VPCs to select",
+      type: { required: true }, // required prop or not
+      control: "none",
+    },
+    subnetList: {
+      description:
+        "An array of strings containing the names of subnet names to choose from",
+      type: { required: true }, // required prop or not
+      control: "none",
+    },
+  },
+  parameters: {
+    docs: {
+      description: {
+        component:
+          "VpnServerForm is a form component that provides functionality for adding or editing a VPN Server.",
+      },
```

Comment:
to fix the modal covering the docs add this here, we need to put the story in an iframe
will go right under `docs: {`

```
      inlineStories: false,
      iframeHeight: 375,
```

Reply:
this fixed it, thank you!